pkg/crc/daemonclient/client.go

```diff
@@ -22,3 +27,19 @@ func New() *Client {
 		}, "http://unix/api"),
 	}
 }
+
+func GetVersionFromDaemonAPI() (*client.VersionResult, error) {
+	daemonClient := New()
+	version, err := daemonClient.APIClient.Version()
+	if err != nil {
+		return nil, pkgerrors.Wrap(err, genericDaemonNotRunningMessage)
```

Comment:
`fmt.Errorf("%w")` can be used to wrap the error instead of the pkg/errors package

Reply:
@anjannath I just moved this piece without change.